src/global.css

```diff
@@ -1,53 +1,8 @@
-html {
-  font-size: 1.125rem;
-  line-height: 1.6;
-  font-weight: 400;
-  font-family: sans-serif;
-  box-sizing: border-box;
-  scroll-behavior: smooth;
-  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+body {
```

Comment:
You can also handle globals with styled-components: https://styled-components.com/docs/api#createglobalstyle